src/js/workshop-comments.js

```diff
@@ -0,0 +1,81 @@
+const form = document.getElementById("comment-form");
+
+const name = document.getElementById("name");
+const comment = document.getElementById("comment");
+
+form.addEventListener("submit", (event) => {
+	event.preventDefault();
+
+	if (setRequiredMessage(name) & setRequiredMessage(comment)) {
+
+		const XHR = new XMLHttpRequest();
+
+		// Bind the FormData object and the form element
+		const FD = new FormData( form );
+
+		// Define what happens on successful data submission
+		XHR.addEventListener( "load", function() {
+			setSubmittedComment();
+		});
+
+		// Define what happens in case of error
+		XHR.addEventListener( "error", function() {
```

Comment:
If the `POST` request to server fails, it's probably better to display the error on the UI so users know what's going on. The displayed message should contain the error returned from the server.

@uttaraghodke, the last page for "Initiatives" shows the successful message when the comment is successfully posted. Could you make a design of how the error should be displayed when the comment is NOT saved successfully. Thanks.